include/evm_runtime/types.hpp

```diff
@@ -18,18 +16,34 @@ namespace evm_runtime {
    static constexpr uint64_t gas_sset_min = 2900;
    static constexpr uint64_t grace_period_seconds = 180;
 
-   constexpr unsigned evm_precision = 18;
-   constexpr eosio::name token_account(eosio::name(TOKEN_ACCOUNT_NAME));
-   constexpr eosio::symbol token_symbol("EOS", 4u);
-   constexpr intx::uint256 minimum_natively_representable = intx::exp(10_u256, intx::uint256(evm_precision - token_symbol.precision()));
-   static_assert(evm_precision - token_symbol.precision() <= 14, "dust math may overflow a uint64_t");
-
-   constexpr double pow10_const(int v) {
-      double r = 1.0;
-      while (v-- > 0) r *= 10.0;
-      return r;
+   inline uint64_t pow10_const(unsigned int v) {
```

Comment:
```c++
template <std::size_t N>
constexpr std::array<intx::uint256, N> generate_N_powers_of_10() {
    std::array<intx::uint256, N> powers{};
    for (auto i=0; i < powers.size(); i++) {
        powers[i] = intx::exp(10_u256, intx::uint256(i));
    }
    return powers;
}

constexpr auto minimum_natively_representable_at = generate_N_powers_of_10<19>();

int main() {
    const auto& minimum_natively_representable = minimum_natively_representable_at[10];
    std::cout << intx::to_string(minimum_natively_representable) << std::endl;
    return 0;
}
```

Reply:
I was thinking that maybe we can generate the uint256 at compile time, and then get a reference from that table everywhere we need the min_native_representable

```
    const auto& minimum_natively_representable = minimum_natively_representable_at[10];
```